tests/dataplane-performance/results/1.0.0.md

```diff
@@ -0,0 +1,123 @@
+# Results
```

Comment:
I think we want a `results/1.0.0` directory to match the pattern of the other tests.